modules/images-imagestream-external-image-tags.adoc

```diff
@@ -10,9 +10,9 @@ You can add tags for external images.
 
 * Add tags pointing to internal or external images, by using the `oc tag` command
  for all tag-related operations:
- +
++
 ----
-oc tag <repositiory/image> <image-name:tag>
+oc tag <image-name:tag1> <image-name:tag2>
 ----
```

Comment:
My suggest: change cmd "oc tag image-name:tag image-name:tag " to "oc tag image-name:tag1 image-name:tag2" in the doc
https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/images-imagestream-adding-tags.adoc

Reply:
Is the most recent correct?

----
oc tag <repository/image> <image-name:tag>
oc tag <image-name:tag1> <image-name:tag2>
----

or  should it be:

----
oc tag <repository/image-name:tag1> <image-name:tag2>
----

Thanks again!

Reply:
"oc tag <repository/image> <image-name:tag>" is correct here, no need change.

I suggest to modify  cmd "oc tag image-name:tag image-name:tag " to "oc tag image-name:tag1 image-name:tag2" in  another doc https://github.com/openshift/openshift-docs/blob/enterprise-4.0/modules/images-imagestream-adding-tags.adoc

Reply:
Thank you!